steps/kotlin-script/1.0.0/step.yml

```diff
@@ -0,0 +1,41 @@
+title: Kotlin Script
+summary: |
+  Step to execute a kotlin script
+description: |
+  Step to execute a kotlin script
+website: https://github.com/jakob-grabner/bitrise-step-kotlin-script
+source_code_url: https://github.com/jakob-grabner/bitrise-step-kotlin-script
+support_url: https://github.com/jakob-grabner/bitrise-step-kotlin-script/issues
+published_at: 2023-10-18T15:02:52.092092+02:00
+source:
+  git: https://github.com/jakob-grabner/bitrise-step-kotlin-script.git
+  commit: 08ea2e8f750f6546c1da89b6387956a59173357a
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
```

Comment:
Kotlin is installed on both macOS and Linux stacks and it's installed via other means, so you can probably remove this deps block and save some unnecessary install time